### PR TITLE
add tuna metrics on support form

### PR DESCRIPTION
**Pull request by grrowe**

### CODE REVIEW

File: src/ui/pages/support.tsx [hunk 1]
```diff
@@ -105,6 +106,13 @@ export const SupportPage = () => {
     e.preventDefault();
     const attachments = attachedFiles.map((file) => file.token);
     dispatch(createSupportTicket({ ...formState, attachments: attachments }));
+    
+    const w = window as any;
```
Comment (neurosnap) [inline src/ui/pages/support.tsx:110]:
```ts
import { tunaEvent } from "@app/tuna";
// ...
tunaEvent('submittedAppUiSupportForm', `{ "viewedDocs": "${viewedSuggestion}", "email": "${user.email}" }`);
```

File: src/ui/pages/support.tsx [hunk 2]
```diff
@@ -236,7 +245,13 @@ export const SupportPage = () => {
                       return (
                         <li key={key + 1}>
                           <div>
-                            <a target="_blank" href={hit.url} rel="noreferrer">
+                            <a target="_blank" href={hit.url} rel="noreferrer" onClick={() => {
+                              const w = window as any;
+                              if (w.aptible?.event) {
+                                  setViewedSuggestion(true)
+                                  w.aptible.event('usedSupportSuggestion', `{ "suggestedUrl": "${hit.url}", "email": "${user.email}" }`);
```
Comment (neurosnap) [inline src/ui/pages/support.tsx:252]: same here

**Review by neurosnap — CHANGES_REQUESTED** [review]
I just recently formalized how we use tuna in `app-ui` which ensures we use it properly.

File: src/ui/pages/support.tsx (hunk 1 above)
Comment (neurosnap) [reply src/ui/pages/support.tsx:110]: I don't think the data needs to be a string either

Comment (grrowe) [reply src/ui/pages/support.tsx]: Needs to be a string.
image
First is as a string, second as an object

File: src/ui/pages/support.tsx [hunk 3]
```diff
@@ -105,6 +107,10 @@ export const SupportPage = () => {
     e.preventDefault();
     const attachments = attachedFiles.map((file) => file.token);
     dispatch(createSupportTicket({ ...formState, attachments: attachments }));
+    tunaEvent(
+      "submittedAppUiSupportForm",
+      `{ "viewedDocs": "${viewedSuggestion}", "email": "${user.email}" }`,
```
Comment (neurosnap) [inline src/ui/pages/support.tsx:112]: `JSON.stringify({ viewedDocs, email })`

Comment (grrowe) [reply src/ui/pages/support.tsx:112]: This encodes with the strange `"{ \"suggestedUrl\": \"https://www.aptible.com/docs/influxdb-metric-drain\", \"email\": \"graham.rowe@aptible.com\" }"` as well as not being able to use JSON selectors (`event_value::json->>'email') in postgres.

Comment (neurosnap) [reply src/ui/pages/support.tsx:112]: got it, thanks for explanation!

**Review by joeclayallday — APPROVED** [review]

**Review by neurosnap — APPROVED** [review]

**Pull request merged**